### PR TITLE
adds install instructions to README

### PR DESCRIPTION
**Pull request by nellshamrell**
Partial fix for #1082 - adds documentation around install Habitat to the README.

Signed-off-by: Nell Shamrell <nellshamrell@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: README.md [hunk 1]
```diff
@@ -22,6 +22,23 @@ applications that:
...
 
+## Installation
+
+There are a couple of different way to install Habitat.
+
+You can download it from the [Habitat downloads page](https://www.habitat.sh/docs/get-habitat/).
+
+Once you have downloaded it, follow the instructions on the page for your specific operating system.
+
+If you are running Linux, you can also use the Habitat install script.
```
Comment (fnichol) [inline README.md]: This script also currently works on Mac as well--it installs the raw `hab` binary to `/usr/local/bin/hab` so if you happen to be a homebrew user, you may even be able to drop the `sudo` call (it works either way though)

Comment (nellshamrell) [inline README.md]: TY!  Will add that in!

**Comment by reset** [comment]
gif-keyboard-2319322617692476051

**Pull request merged**